### DISCUSSION TITLE
Elaboration on FAQ:  is there any kind of rollback?

### DISCUSSION THREAD

**Discussion by castilma** [welaika/wordmove#638] [Q&A]
The current answer to that question is just a general statement.
```
Having backups for each push/pull operations, gives you opportunity to do manual restores.
```

Does wordmove currently create backups for any operation besided database push/pull? I can't find them. Not even in wp-content, which is mentioned in the succeeding question. Or does it refer to the preceeding question, i.e. only the to-be-overwritten database is backed up?

**Comment by alessandro-fazzi** [comment]
It creates backups only for database related operations.
